Tests/GitHubCodespaces.tests.ps1

```diff
@@ -0,0 +1,450 @@
+    }
+    $repo = New-GitHubRepository @newGitHubRepositoryParms
+    # Get the main/master branch name
+    $mainBranchName = $repo | Get-GitHubRepositoryBranch | Select-Object -ExpandProperty name -First 1
+}
+
+Describe 'GitHubCodespaces\Delete-GitHubCodespace' {
+    Context 'When deleting a codespace for the authenticated user' {
+        BeforeEach {
+            # Suppress HTTP 202 warning for codespace creation
+            # TODO: Suppression is not working as intended here.
+            $WarningPreference = 'SilentlyContinue'
+
+            $newGitHubCodespaceParms = @{
+                OwnerName = $script:organizationName
+                RepositoryName = $defaultRepositoryName
+            }
+            $codespace = New-GitHubCodespace @newGitHubCodespaceParms -Wait
+        }
+
+        It 'Should get no content using -Confirm:$false' {
+            # Also asserts pipeline input
+            $codespace | Remove-GitHubCodespace -Confirm:$false
+            { Get-GitHubCodespace -CodespaceName $codespace.name } | Should -Throw
+        }
+
+        It 'Should get no content using -Force' {
+            # Also assert CodespaceName input
+            Remove-GitHubCodespace -CodespaceName $codespace.name -Force
+            { Get-GitHubCodespace -CodespaceName $codespace.name } | Should -Throw
+        }
+    }
+}
+
+Describe 'GitHubCodespaces\Get-GitHubCodespace' {
+    BeforeAll {
+        # Suppress HTTP 202 warning for codespace creation
+        $WarningPreference = 'SilentlyContinue'
+
+        $newGitHubCodespaceParms = @{
+            OwnerName = $script:organizationName
+            RepositoryName = $defaultRepositoryName
+        }
+        $null = New-GitHubCodespace @newGitHubCodespaceParms -Wait
+    }
+
+    Context 'When getting codespaces for the authenticated user' {
+        BeforeAll {
+            $codespaces = Get-GitHubCodespace |
+                Where-Object { $_.repository.name -eq $defaultRepositoryName }
+        }
+
+        It 'Should return objects of the correct type' {
+            $codespaces[0].PSObject.TypeNames[0] | Should -Be 'GitHub.Codespace'
+        }
+
+        It 'Should return one or more results' {
+            ($codespaces | Where-Object { $_ }).Count | Should -BeGreaterOrEqual 1
+        }
+
+        It 'Should return the correct properties' {
+            foreach ($codespace in $codespaces)
+            {
+                $codespace.owner.login | Should -Be $script:OwnerName
+                $codespace.repository.name | Should -Be $newGitHubCodespaceParms.RepositoryName
+            }
+        }
+    }
+
+    Context 'When getting a codespace for a specified owner and repository' {
+        BeforeAll {
+            $codespaces = Get-GitHubCodespace @newGitHubCodespaceParms
+        }
+
+        It 'Should return objects of the correct type' {
+            $codespaces[0].PSObject.TypeNames[0] | Should -Be 'GitHub.Codespace'
+        }
+
+        It 'Should return one or more results' {
```

Comment:
``
[-] GitHubCodespaces\Get-GitHubCodespace.When getting a codespace for a specified owner and repository.Should return one or more results 13ms (11ms|2ms)
 Expected the actual value to be greater than or equal to 1, but got $null.
 at ($codespaces | Where-Object { $_ }).Count | Should -BeGreaterOrEqual 1, C:\Source\PowerShellForGitHub\tests\GitHubCodespaces.tests.ps1:110
 at <ScriptBlock>, C:\Source\PowerShellForGitHub\tests\GitHubCodespaces.tests.ps1:110
``

Reply:
Should be resolved now with `OwnerName = $repo.owner.login`.